tests/utils/test_email.py

```diff
@@ -273,7 +273,7 @@ def test_send_mime_dryrun(self, mock_smtp, mock_smtp_ssl):
 
     @mock.patch('smtplib.SMTP_SSL')
     @mock.patch('smtplib.SMTP')
-    def test_send_mime_complete_failure(self, mock_smtp: mock, mock_smtp_ssl):
+    def test_send_mime_complete_failure(self, mock_smtp, mock_smtp_ssl):
```

Comment:
Let’s annotate it correctly instead?

```suggestion
    def test_send_mime_complete_failure(self, mock_smtp: mock.Mock, mock_smtp_ssl: mock.Mock):
```